### ISSUE TITLE
Cannot send mail if subject contains non-ascii characters

### ISSUE THREAD

**Issue by rabelux** [openmediavault/openmediavault#1058]
**Describe the bug**
If an rsync job fails due to loss of internet connection, there is no mail notification and no entry in `rsync.log`.
In `syslog` there is a line containing `Jun 30 04:56:26 NanoPi rsync-2d15bb93-ecb8-46f6-87c1-b23ceb4870ef: The synchronisation failed.` but I think the better place to inform me about a failed job is `rsync.log`

**Expected behavior**
Either an email is sent after the connection is restored telling me about the failed job, or rsync log contains a line telling me about the failure.

**openmediavault Server:**

- OS version: Linux NanoPi 5.10.43-rockchip64
- openmediavault version: 5.6.11

**Additional context**
I'm running a remote backup once a week that takes 10-20 hours and sometimes the source/target resets their internet connection within this timeframe. Currently I can only tell something went wrong by *not* receiving an email.

***Question:***
Is it intended that (rsync-)jobs started in the GUI do not send mails?
As I have to restart a job if it failed using the GUI a notification would be welcomed.

**Comment by votdev** [comment]
According to https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault/srv/salt/omv/deploy/rsync/files/cron-rsync.j2#L19 stderr is redirected to stdout, so if rsync writes an error to stderr it should be in the email. But i don't know the behaviour of cron when the executed command exits with an error code. That is something to be investigated.
I guess the problem is the [trap](https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault/srv/salt/omv/deploy/rsync/files/cron-rsync-script.j2#L27) in the script that is executed by cron.
And contribution is welcome to get this issue implemented.

**Comment by votdev** [comment]
OMV already prints out an [error message to stderr](https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault/srv/salt/omv/deploy/rsync/files/cron-rsync-script.j2#L82) if rsync exists != 0. No idea why the email is not sent, maybe cron is the reason?

Someone interested to take over and digg into this issue?

**Comment by votdev** [comment]
Played a little bit with the script to emulate a problem with rsync and got an email.
```
Please wait, syncing </srv/dev-disk-by-uuid-39698cf2-54c1-4021-8446-29365b6df64a/source/> to <root@172.72.16.10:test> ...

The --contimeout option may only be used when connecting to an rsync daemon.
rsync error: syntax or usage error (code 1) at main.c(1546) [sender=3.2.3]
ERROR: The synchronisation failed.
```
Because of that i assume an email should be send whenever there is a problem with rsync. I think you should investigate that issue on your system because i can not reproduce it.

**Comment by rabelux** [comment]
Thanks for digging a bit!
Please answer me one question:
Is it intended that (rsync-)jobs started in the GUI do not send mails?

Because they don't and it would make it easier for me to narrow down the problem.

**Comment by votdev** [comment]
> Thanks for digging a bit!
> Please answer me one question:
> Is it intended that (rsync-)jobs started in the GUI do not send mails?
> 
> Because they don't ~and it would make it easier for me to narrow down the problem~.

Yes, because the command output is displayed there.

> EDIT:
> Found out I can run the line in `/etc/cron.d/openmediavault-rsync` to trigger sending an email.
> 
> Will report back.

**Comment by rabelux** [comment]
I can reproduce exactly what I wrote in the beginning of this ticket.
My observed behaviour is that:
- if rsync fails straight away a message is sent
- if rsync fails somewhere midterm (due to reconnect, that's what I'm testing) no mail is sent

Can I reproduce

**Comment by votdev** [comment]
>     * if rsync fails somewhere midterm (due to reconnect, that's what I'm testing) no mail is sent

I have no idea what the reason could be. Rather unlikely, but may it happen that rsync still runs and never returns? If it never exits, then no email is sent.

**Comment by rabelux** [comment]
The command does return and `echo $?` yields 0.
This is the command I ran: `sudo /var/lib/openmediavault/cron.d/rsync-2d15bb93-ecb8-46f6-87c1-b23ceb4870ef 2>&1 | mail -E -s "Cron - Medien" -a "From: Cron Daemon <root>" root >/dev/null 2>&1`

Ignoring the mail-problem for a second, would it be possible to include the error-message in *rsync.log*? Currently it only appears in *syslog*: `Jun 30 04:56:26 NanoPi rsync-2d15bb93-ecb8-46f6-87c1-b23ceb4870ef: The synchronisation failed.`

Back to the mail problem, I realized syslog gets spammed with these messages after losing connection (connection is restored after ~10s):
<details>
<summary>Click to expand</summary>

```
Jul  1 17:05:12 NanoPi rsync-2d15bb93-ecb8-46f6-87c1-b23ceb4870ef: The synchronisation failed.
Jul  1 17:05:12 NanoPi postfix/pickup[31392]: 4A30C10A: uid=1000 from=<raab>
Jul  1 17:05:12 NanoPi postfix/cleanup[4852]: 4A30C10A: message-id=<20210701150512.4A30C10A@NanoPi.localdomain>
Jul  1 17:05:12 NanoPi postfix/qmgr[2302]: 4A30C10A: from=<MailSender@gmx.de>, size=1377, nrcpt=2 (queue active)
Jul  1 17:05:12 NanoPi postfix/pipe[4869]: 4A30C10A: to=<openmediavault-notification@localhost.localdomain>, relay=omvnotificationfilter, delay=0.13, delays=0.06/0.02/0/0.05, dsn=2.0.0, status=sent (delivered via omvnotificationfilter service)
Jul  1 17:05:12 NanoPi postfix/smtp[4870]: 4A30C10A: to=<MailReceiver@gmx.de>, orig_to=<root>, relay=mail.gmx.net[212.227.17.190]:587, delay=0.24, delays=0.06/0.05/0.13/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:12 NanoPi postfix/cleanup[4852]: 82F9810C: message-id=<20210701150512.82F9810C@NanoPi.localdomain>
Jul  1 17:05:12 NanoPi postfix/bounce[4881]: 4A30C10A: sender non-delivery notification: 82F9810C
Jul  1 17:05:12 NanoPi postfix/qmgr[2302]: 82F9810C: from=<MailSender@gmx.de>, size=3429, nrcpt=1 (queue active)
Jul  1 17:05:12 NanoPi postfix/qmgr[2302]: 4A30C10A: removed
Jul  1 17:05:13 NanoPi postfix/smtp[4870]: 82F9810C: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.2, delays=0/1/0.16/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:13 NanoPi postfix/cleanup[4852]: AC0A910E: message-id=<20210701150513.AC0A910E@NanoPi.localdomain>
Jul  1 17:05:13 NanoPi postfix/bounce[4881]: 82F9810C: sender non-delivery notification: AC0A910E
Jul  1 17:05:13 NanoPi postfix/qmgr[2302]: AC0A910E: from=<MailSender@gmx.de>, size=5446, nrcpt=1 (queue active)
Jul  1 17:05:13 NanoPi postfix/qmgr[2302]: 82F9810C: removed
Jul  1 17:05:14 NanoPi postfix/smtp[4870]: AC0A910E: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:14 NanoPi postfix/cleanup[4852]: C4B0F110: message-id=<20210701150514.C4B0F110@NanoPi.localdomain>
Jul  1 17:05:14 NanoPi postfix/bounce[4881]: AC0A910E: sender non-delivery notification: C4B0F110
Jul  1 17:05:14 NanoPi postfix/qmgr[2302]: C4B0F110: from=<MailSender@gmx.de>, size=7463, nrcpt=1 (queue active)
Jul  1 17:05:14 NanoPi postfix/qmgr[2302]: AC0A910E: removed
Jul  1 17:05:15 NanoPi postfix/smtp[4870]: C4B0F110: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:15 NanoPi postfix/cleanup[4852]: DE4A0112: message-id=<20210701150515.DE4A0112@NanoPi.localdomain>
Jul  1 17:05:15 NanoPi postfix/bounce[4881]: C4B0F110: sender non-delivery notification: DE4A0112
Jul  1 17:05:15 NanoPi postfix/qmgr[2302]: DE4A0112: from=<MailSender@gmx.de>, size=9480, nrcpt=1 (queue active)
Jul  1 17:05:15 NanoPi postfix/qmgr[2302]: C4B0F110: removed
Jul  1 17:05:17 NanoPi postfix/smtp[4870]: DE4A0112: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0.01/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:17 NanoPi postfix/cleanup[4852]: 03B29114: message-id=<20210701150517.03B29114@NanoPi.localdomain>
Jul  1 17:05:17 NanoPi postfix/bounce[4881]: DE4A0112: sender non-delivery notification: 03B29114
Jul  1 17:05:17 NanoPi postfix/qmgr[2302]: 03B29114: from=<MailSender@gmx.de>, size=11497, nrcpt=1 (queue active)
Jul  1 17:05:17 NanoPi postfix/qmgr[2302]: DE4A0112: removed
Jul  1 17:05:18 NanoPi postfix/smtp[4870]: 03B29114: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:18 NanoPi postfix/cleanup[4852]: 1C34A116: message-id=<20210701150518.1C34A116@NanoPi.localdomain>
Jul  1 17:05:18 NanoPi postfix/qmgr[2302]: 1C34A116: from=<MailSender@gmx.de>, size=13514, nrcpt=1 (queue active)
Jul  1 17:05:18 NanoPi postfix/bounce[4881]: 03B29114: sender non-delivery notification: 1C34A116
Jul  1 17:05:18 NanoPi postfix/qmgr[2302]: 03B29114: removed
Jul  1 17:05:19 NanoPi postfix/smtp[4870]: 1C34A116: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0/1/0.1/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:19 NanoPi postfix/cleanup[4852]: 34F21118: message-id=<20210701150519.34F21118@NanoPi.localdomain>
Jul  1 17:05:19 NanoPi postfix/bounce[4881]: 1C34A116: sender non-delivery notification: 34F21118
Jul  1 17:05:19 NanoPi postfix/qmgr[2302]: 34F21118: from=<MailSender@gmx.de>, size=15531, nrcpt=1 (queue active)
Jul  1 17:05:19 NanoPi postfix/qmgr[2302]: 1C34A116: removed
Jul  1 17:05:20 NanoPi postfix/smtp[4870]: 34F21118: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:20 NanoPi postfix/cleanup[4852]: 4DC9911A: message-id=<20210701150520.4DC9911A@NanoPi.localdomain>
Jul  1 17:05:20 NanoPi postfix/qmgr[2302]: 4DC9911A: from=<MailSender@gmx.de>, size=17548, nrcpt=1 (queue active)
Jul  1 17:05:20 NanoPi postfix/bounce[4881]: 34F21118: sender non-delivery notification: 4DC9911A
Jul  1 17:05:20 NanoPi postfix/qmgr[2302]: 34F21118: removed
Jul  1 17:05:21 NanoPi postfix/smtp[4870]: 4DC9911A: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:21 NanoPi postfix/cleanup[4852]: 64CC211C: message-id=<20210701150521.64CC211C@NanoPi.localdomain>
Jul  1 17:05:21 NanoPi postfix/bounce[4881]: 4DC9911A: sender non-delivery notification: 64CC211C
Jul  1 17:05:21 NanoPi postfix/qmgr[2302]: 64CC211C: from=<MailSender@gmx.de>, size=19565, nrcpt=1 (queue active)
Jul  1 17:05:21 NanoPi postfix/qmgr[2302]: 4DC9911A: removed
Jul  1 17:05:22 NanoPi postfix/smtp[4870]: 64CC211C: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:22 NanoPi postfix/cleanup[4852]: 7BA1F11E: message-id=<20210701150522.7BA1F11E@NanoPi.localdomain>
Jul  1 17:05:22 NanoPi postfix/bounce[4881]: 64CC211C: sender non-delivery notification: 7BA1F11E
Jul  1 17:05:22 NanoPi postfix/qmgr[2302]: 7BA1F11E: from=<MailSender@gmx.de>, size=21582, nrcpt=1 (queue active)
Jul  1 17:05:22 NanoPi postfix/qmgr[2302]: 64CC211C: removed
Jul  1 17:05:23 NanoPi postfix/smtp[4870]: 7BA1F11E: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:23 NanoPi postfix/cleanup[4852]: 9330F120: message-id=<20210701150523.9330F120@NanoPi.localdomain>
Jul  1 17:05:23 NanoPi postfix/bounce[4881]: 7BA1F11E: sender non-delivery notification: 9330F120
Jul  1 17:05:23 NanoPi postfix/qmgr[2302]: 9330F120: from=<MailSender@gmx.de>, size=23599, nrcpt=1 (queue active)
Jul  1 17:05:23 NanoPi postfix/qmgr[2302]: 7BA1F11E: removed
Jul  1 17:05:24 NanoPi postfix/smtp[4870]: 9330F120: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:24 NanoPi postfix/cleanup[4852]: AB716122: message-id=<20210701150524.AB716122@NanoPi.localdomain>
Jul  1 17:05:24 NanoPi postfix/qmgr[2302]: AB716122: from=<MailSender@gmx.de>, size=25616, nrcpt=1 (queue active)
Jul  1 17:05:24 NanoPi postfix/bounce[4881]: 9330F120: sender non-delivery notification: AB716122
Jul  1 17:05:24 NanoPi postfix/qmgr[2302]: 9330F120: removed
Jul  1 17:05:25 NanoPi postfix/smtp[4870]: AB716122: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0/1/0.1/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:25 NanoPi postfix/cleanup[4852]: C4DE9124: message-id=<20210701150525.C4DE9124@NanoPi.localdomain>
Jul  1 17:05:25 NanoPi postfix/bounce[4881]: AB716122: sender non-delivery notification: C4DE9124
Jul  1 17:05:25 NanoPi postfix/qmgr[2302]: C4DE9124: from=<MailSender@gmx.de>, size=27633, nrcpt=1 (queue active)
Jul  1 17:05:25 NanoPi postfix/qmgr[2302]: AB716122: removed
Jul  1 17:05:26 NanoPi postfix/smtp[4870]: C4DE9124: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0.01/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:26 NanoPi postfix/cleanup[4852]: DD598126: message-id=<20210701150526.DD598126@NanoPi.localdomain>
Jul  1 17:05:26 NanoPi postfix/bounce[4881]: C4DE9124: sender non-delivery notification: DD598126
Jul  1 17:05:26 NanoPi postfix/qmgr[2302]: DD598126: from=<MailSender@gmx.de>, size=29650, nrcpt=1 (queue active)
Jul  1 17:05:26 NanoPi postfix/qmgr[2302]: C4DE9124: removed
Jul  1 17:05:28 NanoPi postfix/smtp[4870]: DD598126: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0.01/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:28 NanoPi postfix/cleanup[4852]: 01A0D128: message-id=<20210701150528.01A0D128@NanoPi.localdomain>
Jul  1 17:05:28 NanoPi postfix/bounce[4881]: DD598126: sender non-delivery notification: 01A0D128
Jul  1 17:05:28 NanoPi postfix/qmgr[2302]: 01A0D128: from=<MailSender@gmx.de>, size=31667, nrcpt=1 (queue active)
Jul  1 17:05:28 NanoPi postfix/qmgr[2302]: DD598126: removed
Jul  1 17:05:29 NanoPi postfix/smtp[4870]: 01A0D128: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0.01/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:29 NanoPi postfix/cleanup[4852]: 1AEA312A: message-id=<20210701150529.1AEA312A@NanoPi.localdomain>
Jul  1 17:05:29 NanoPi postfix/bounce[4881]: 01A0D128: sender non-delivery notification: 1AEA312A
Jul  1 17:05:29 NanoPi postfix/qmgr[2302]: 1AEA312A: from=<MailSender@gmx.de>, size=33684, nrcpt=1 (queue active)
Jul  1 17:05:29 NanoPi postfix/qmgr[2302]: 01A0D128: removed
Jul  1 17:05:30 NanoPi postfix/smtp[4870]: 1AEA312A: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0.01/1/0.1/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:30 NanoPi postfix/cleanup[4852]: 3590C12C: message-id=<20210701150530.3590C12C@NanoPi.localdomain>
Jul  1 17:05:30 NanoPi postfix/qmgr[2302]: 3590C12C: from=<MailSender@gmx.de>, size=35701, nrcpt=1 (queue active)
Jul  1 17:05:30 NanoPi postfix/bounce[4881]: 1AEA312A: sender non-delivery notification: 3590C12C
Jul  1 17:05:30 NanoPi postfix/qmgr[2302]: 1AEA312A: removed
Jul  1 17:05:31 NanoPi postfix/smtp[4870]: 3590C12C: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0/1/0.1/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:31 NanoPi postfix/cleanup[4852]: 4FBA312E: message-id=<20210701150531.4FBA312E@NanoPi.localdomain>
Jul  1 17:05:31 NanoPi postfix/bounce[4881]: 3590C12C: sender non-delivery notification: 4FBA312E
Jul  1 17:05:31 NanoPi postfix/qmgr[2302]: 4FBA312E: from=<MailSender@gmx.de>, size=37718, nrcpt=1 (queue active)
Jul  1 17:05:31 NanoPi postfix/qmgr[2302]: 3590C12C: removed
Jul  1 17:05:32 NanoPi postfix/smtp[4870]: 4FBA312E: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0.01/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:32 NanoPi postfix/cleanup[4852]: 69655130: message-id=<20210701150532.69655130@NanoPi.localdomain>
Jul  1 17:05:32 NanoPi postfix/bounce[4881]: 4FBA312E: sender non-delivery notification: 69655130
Jul  1 17:05:32 NanoPi postfix/qmgr[2302]: 69655130: from=<MailSender@gmx.de>, size=39735, nrcpt=1 (queue active)
Jul  1 17:05:32 NanoPi postfix/qmgr[2302]: 4FBA312E: removed
Jul  1 17:05:33 NanoPi postfix/smtp[4870]: 69655130: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:33 NanoPi postfix/cleanup[4852]: 81071132: message-id=<20210701150533.81071132@NanoPi.localdomain>
Jul  1 17:05:33 NanoPi postfix/bounce[4881]: 69655130: sender non-delivery notification: 81071132
Jul  1 17:05:33 NanoPi postfix/qmgr[2302]: 81071132: from=<MailSender@gmx.de>, size=41752, nrcpt=1 (queue active)
Jul  1 17:05:33 NanoPi postfix/qmgr[2302]: 69655130: removed
Jul  1 17:05:34 NanoPi postfix/smtp[4870]: 81071132: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:34 NanoPi postfix/cleanup[4852]: 9A215134: message-id=<20210701150534.9A215134@NanoPi.localdomain>
Jul  1 17:05:34 NanoPi postfix/bounce[4881]: 81071132: sender non-delivery notification: 9A215134
Jul  1 17:05:34 NanoPi postfix/qmgr[2302]: 9A215134: from=<MailSender@gmx.de>, size=43769, nrcpt=1 (queue active)
Jul  1 17:05:34 NanoPi postfix/qmgr[2302]: 81071132: removed
Jul  1 17:05:35 NanoPi postfix/smtp[4870]: 9A215134: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0.01/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:35 NanoPi postfix/cleanup[4852]: B4BED136: message-id=<20210701150535.B4BED136@NanoPi.localdomain>
Jul  1 17:05:35 NanoPi postfix/bounce[4881]: 9A215134: sender non-delivery notification: B4BED136
Jul  1 17:05:35 NanoPi postfix/qmgr[2302]: B4BED136: from=<MailSender@gmx.de>, size=45786, nrcpt=1 (queue active)
Jul  1 17:05:35 NanoPi postfix/qmgr[2302]: 9A215134: removed
Jul  1 17:05:36 NanoPi postfix/smtp[4870]: B4BED136: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:36 NanoPi postfix/cleanup[4852]: CD5A3138: message-id=<20210701150536.CD5A3138@NanoPi.localdomain>
Jul  1 17:05:36 NanoPi postfix/bounce[4881]: B4BED136: sender non-delivery notification: CD5A3138
Jul  1 17:05:36 NanoPi postfix/qmgr[2302]: CD5A3138: from=<MailSender@gmx.de>, size=47803, nrcpt=1 (queue active)
Jul  1 17:05:36 NanoPi postfix/qmgr[2302]: B4BED136: removed
Jul  1 17:05:37 NanoPi postfix/smtp[4870]: CD5A3138: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.168]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.168])
Jul  1 17:05:37 NanoPi postfix/cleanup[4852]: E659F13A: message-id=<20210701150537.E659F13A@NanoPi.localdomain>
Jul  1 17:05:37 NanoPi postfix/bounce[4881]: CD5A3138: sender non-delivery notification: E659F13A
Jul  1 17:05:37 NanoPi postfix/qmgr[2302]: E659F13A: from=<MailSender@gmx.de>, size=49820, nrcpt=1 (queue active)
Jul  1 17:05:37 NanoPi postfix/qmgr[2302]: CD5A3138: removed
Jul  1 17:05:39 NanoPi postfix/smtp[4870]: E659F13A: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0.01/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:39 NanoPi postfix/cleanup[4852]: 0D3C813C: message-id=<20210701150539.0D3C813C@NanoPi.localdomain>
Jul  1 17:05:39 NanoPi postfix/bounce[4881]: E659F13A: sender non-delivery notification: 0D3C813C
Jul  1 17:05:39 NanoPi postfix/qmgr[2302]: 0D3C813C: from=<MailSender@gmx.de>, size=51837, nrcpt=1 (queue active)
Jul  1 17:05:39 NanoPi postfix/qmgr[2302]: E659F13A: removed
Jul  1 17:05:40 NanoPi postfix/smtp[4870]: 0D3C813C: to=<MailSender@gmx.de>, relay=mail.gmx.net[212.227.17.190]:587, delay=1.1, delays=0/1/0.09/0, dsn=5.6.7, status=bounced (SMTPUTF8 is required, but was not offered by host mail.gmx.net[212.227.17.190])
Jul  1 17:05:40 NanoPi postfix/cleanup[4852]: 2623613E: message-id=<20210701150540.2623613E@NanoPi.localdomain>
Jul  1 17:05:40 NanoPi postfix/bounce[4881]: 0D3C813C: sender non-delivery notification: 2623613E
Jul  1 17:05:40 NanoPi postfix/qmgr[2302]: 2623613E: from=<MailSender@gmx.de>, size=2549, nrcpt=1 (queue active)
Jul  1 17:05:40 NanoPi postfix/qmgr[2302]: 0D3C813C: removed
Jul  1 17:05:41 NanoPi postfix/smtp[4870]: 2623613E: replace: header Subject: Undelivered Mail Returned to Sender: Subject: [NanoPi] Undelivered Mail Returned to Sender
Jul  1 17:05:41 NanoPi postfix/qmgr[2302]: 2623613E: removed
Jul  1 17:07:26 NanoPi postfix/scache[4883]: statistics: start interval Jul  1 17:05:13
Jul  1 17:07:26 NanoPi postfix/scache[4883]: statistics: domain lookup hits=0 miss=26 success=0%
Jul  1 17:07:26 NanoPi postfix/scache[4883]: statistics: address lookup hits=0 miss=52 success=0%
```
</details>

which looks like a setup-problem to me, but other rsync jobs are able to send mails.
I'll try to finish the current rsync job to see if I receive an email.

**Comment by rabelux** [comment]
This is what a successful send-action looks like:

```
Jul  1 01:00:16 NanoPi rsync-4a9bead5-8944-4b32-bfe3-6fcabc840f4a: The synchronisation has completed successfully.
Jul  1 01:00:16 NanoPi postfix/pickup[21808]: ECF43D0: uid=0 from=<root>
Jul  1 01:00:17 NanoPi postfix/cleanup[3317]: ECF43D0: message-id=<20210630230016.ECF43D0@NanoPi.localdomain>
Jul  1 01:00:17 NanoPi postfix/qmgr[2302]: ECF43D0: from=<MailSender@gmx.de>, size=1229, nrcpt=2 (queue active)
Jul  1 01:00:17 NanoPi postfix/pipe[3319]: ECF43D0: to=<openmediavault-notification@localhost.localdomain>, relay=omvnotificationfilter, delay=0.16, delays=0.08/0.03/0/0.05, dsn=2.0.0, status=sent (delivered via omvnotificationfilter service)
Jul  1 01:00:17 NanoPi postfix/smtp[3320]: ECF43D0: replace: header Subject: Cron - Foto-Sicherung: Subject: [NanoPi] Cron - Foto-Sicherung
Jul  1 01:00:17 NanoPi postfix/qmgr[2302]: ECF43D0: removed
```

And I've found non-delivery reports in my sender's spam folder.

**Comment by rabelux** [comment]
Had some time to dig into this issue and found out that the root cause is something completely different.
### TL;DR:
If the header of an email contains non-ascii characters, postfix tries to send it as utf-8 <sup>[1]</sup>.
Apparently my mail-provider doesn't support this so to prevent utf-8-encoding I added `smtputf8_enable=no` to */etc/postfix/main.cf* and reloaded with `postfix reload`<sup>[2]</sup>.
### Background:
Out of my 4 rsync jobs (configured in web-gui) the longest one (which I set up 3 weeks ago and which always got interrupted by a disconnect so far) never sent a notification mail. It was also the only one that had an Umlaut in its comment and that was the root of the issue.
### Solution:
I thought about opening a pull-request in [main.cf.j2](https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault/srv/salt/omv/deploy/postfix/files/main.cf.j2) to add `smtputf8_enable=no` but I don't know whether you want to introduce a parameter for it. As apparently all mails are being sent as ascii so far I can't think of a downside if this line as to be added. MS Outlook handles the char-conversion as desired:
![grafik](https://user-images.githubusercontent.com/16120882/124386160-5eb70a00-dcd9-11eb-8896-edfa2ec9b891.png)



[1]: https://qastack.com.de/unix/195381/how-to-prevent-bounce-of-smtputf8-emails
[2]: https://it-muecke.de/node:20201214-postfix-smtputf8-error

**Comment by votdev** [comment]
@rabelux Thanks for digging into this. It's weird that UTF8 is still causing problems like this these days

**Comment by rabelux** [comment]
No problem.
I have to thank *you* for developing OMV!!
